common/app/model/dotcomrendering/DotcomRenderingDataModel.scala

```diff
@@ -283,6 +290,7 @@ object DotcomRenderingDataModel {
       pinnedPost: Option[APIBlock],
       keyEvents: Seq[APIBlock],
       filterKeyEvents: Boolean = false,
+      mostRecentBlockId: Option[String] = None,
```

Comment:
Similar to my comment above, is this optional based on the type of article or could you have a blog that doesn't have this property set?